GVRf/Framework/src/org/gearvrf/GVRContext.java

```diff
@@ -899,4 +899,41 @@ protected synchronized static void resetOnRestart() {
     abstract GVRRenderBundle getRenderBundle();
 
     abstract GVRRecyclableObjectProtector getRecyclableObjectProtector();
+
+    /**
+     * Capture a 2D screenshot from the position in the middle of left eye and
+     * right eye.
+     * 
+     * The screenshot capture is done asynchronously -- the function does not
+     * return the result immediately. Instead, it registers a callback function
+     * and pass the result (when it is available) to the callback function.
+     * 
+     * @param callback
+     *            handle a the callback function to process the capture result
```

Comment:
I think we could probably lose the "handle a the" part. You probably should mention that this is a required parameter, that may not be `null`.